plugins/action/action_impl.cpp

```diff
@@ -411,21 +411,18 @@ void ActionImpl::set_takeoff_altitude(float relative_altitude_m)
 void ActionImpl::receive_takeoff_alt_param(bool success, float new_relative_altitude_m)
 {
     if (success) {
-        // TODO: This should not be buffered like this, the param system
-        //       needs some refactoring.
         _relative_takeoff_altitude_m = new_relative_altitude_m;
     }
 }
 
 float ActionImpl::get_takeoff_altitude_m() const
 {
+    // FIXME: we never actually get the param, as we should.
```

Comment:
So that means that you need to set it from the user side in order to be able to get it?

Reply:
Exactly. I'll refactor the param interface soon, see #440.